examples/design-system-tutorial/src/components/card/card.ts

```diff
@@ -0,0 +1,17 @@
+import { Card, FoundationElementDefinition } from "@microsoft/fast-foundation";
+import { ViewTemplate } from "@microsoft/fast-element";
+
+/**
+ * We can provide custom configuration for a component by defining a type as
+ * shown below and then using it in template or style factories along with the
+ * call to FoundationElement.compose.
+ */
+export type DemoCardDefinitionOptions = FoundationElementDefinition & {
+    likeButton?: ViewTemplate;
+};
+
+/**
```

Comment:
The recipe example doesn't work because Card doesn't use `neutralFillLayer` anywhere to derive it's fill color.

You'll need to change this definition to something like:

```ts
export class DemoCard extends Card {
    connectedCallback(): void {
        super.connectedCallback();

        const { parentElement } = this;

        if (parentElement instanceof HTMLElement) {
            fillColor.setValueFor(this, (source: HTMLElement) =>
                neutralFillLayerRecipe
                    .getValueFor(this)
                    .evaluate(this, fillColor.getValueFor(parentElement))
            );
        }
    }
}
```

I would probably change them both to use `neutralLayerFloating` and `neutralLayerFloatingRecipe` respectively, I think that is the recipe card's are using in the fluent system. @bheston to confirm.

Reply:
Cards use `neutralFillLayerRecipe` by default. Floating is used for dialogs or flyouts. Note that this recipe is _not_ one of the fixed `neutralLayer...` (no "Fill") recipes which work based on `baseLayerLuminance`.